From 6cc4fa914fbd3ed233fe50a8182c724dfe1889e3 Mon Sep 17 00:00:00 2001
From: Bryce Poole <Bryce.Poole@intel.com>
Date: Wed, 9 Dec 2015 15:12:21 -0800
Subject: [PATCH] Enabling Power Management for SDHCI/eMMC/SD Card

Re-enabling PM that was temporarily disabled to allow BIOS to add a
power 'mediator' that disabled eMMC only if both the processors request
it to be powered off or either request power on.
---
 drivers/mmc/pm.h |    2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/mmc/pm.h b/drivers/mmc/pm.h
index eea4b44..87d6249 100644
--- a/drivers/mmc/pm.h
+++ b/drivers/mmc/pm.h
@@ -1,5 +1,3 @@
-#undef CONFIG_PM_RUNTIME
-
 #if 1
 #define mDBG(M, ...) {}
 #define mDBG_mmc_host(mmc_host) {}
-- 
1.7.9.5

